src/SamplesApp/SamplesApp.UITests/Windows_UI_Core/SystemNavigationManagerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+
+namespace SamplesApp.UITests.Windows_UI_Core
+{
+	[TestFixture]
+	public partial class SystemNavigationManagerTests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void When_Hardware_Back_Button_Pressed()
```

Comment:
This should be Android-specific, or maybe android+browser.

Reply:
Done 👍